README.md

```diff
@@ -1,3 +1,39 @@
 # RHMAP Nagios for Docker
 
 Nagios server in a Docker container for RHMAP.
+
+# Setting up nagios service account
+
+In order to talk to the OpenShift and perform exec commands on pods, a service account will need to be created for use by nagios and given the proper permissions.
+
+1. Create the account using oc in the same project as the mbaas/core you are monitoring:
+
+```
+cat > nagios-sa.yaml << EOF
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nagios
+EOF
+ 
+oc create -f nagios-sa.yaml
+sudo oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:<rhmap-project>:nagios
+oc policy add-role-to-user admin system:serviceaccount:<rhmap-project>:nagios
+```
+
+2. Edit the nagios deployment configuration to use the nagios service account by adding the serviceAccount and serviceAccountName under spec.template.spec:
```

Comment:
There's no release yet, so there should be no cluster in that situation.
If that was the case, then this step should be prefixed with the same disclaimer you gave, "This is only needed if ...".

Since it's a very vague step based on `oc edit`, and not necessary for folks installing core today, I'd just get rid of it.